irohad/torii/impl/command_service_transport_grpc.hpp

```diff
@@ -134,6 +134,7 @@ namespace torii {
 
     rxcpp::observable<ConsensusGateEvent> consensus_gate_objects_;
     const int maximum_rounds_without_update_;
+    std::mutex stream_write_mutex_;
```

Comment:
This should be a local variable in `StatusStream`, since writes to each client should be synchronized.